packages/babel-plugin/src/visitors/stylex-create/parse-stylex-create-arg.js

```diff
@@ -77,9 +78,21 @@ export function evaluateStyleXCreateArg(
         pathUtils.isIdentifier(param),
       )
       .map((param) => param.node);
+
+    if (allParams.some((param) => pathUtils.isObjectPattern(param))) {
+      throw new Error(
+        messages.ONLY_NAMED_PARAMETERS_IN_DYNAMIC_STYLE_FUNCTIONS,
+      );
+    }
+
+    if (allParams.some((param) => pathUtils.isAssignmentPattern(param))) {
+      throw new Error(messages.NO_DYNAMIC_STYLE_DEFAULT_PARAMETERS);
+    }
+
     if (params.length !== allParams.length) {
```

Comment:
I think this check is obsolete at this point. can't think of other corner cases besides ones covered above

Reply:
You're right. Only simple named arguments are supported.